<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WINDOW" failures="1" tests="2" name="Login Tests" time="34.835" errors="0" timestamp="2025-09-15T17:21:05 IST">
  <testcase classname="com.orangehrm.LoginTest" name="loginDataDrivenTest" time="25.741">
    <failure type="java.lang.AssertionError" message="Exception during login for: Adm">
      <![CDATA[java.lang.AssertionError: Exception during login for: Adm
at com.orangehrm.LoginTest.loginDataDrivenTest(LoginTest.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- loginDataDrivenTest -->
  <testcase classname="com.orangehrm.LoginTest" name="loginDataDrivenTest" time="5.307"/>
</testsuite> <!-- Login Tests -->
